nflows/transforms/standard.py

```diff
@@ -1,82 +1,92 @@
 """Implementations of some standard transforms."""
 
+from typing import Optional
+import warnings
+import numpy as np
 import torch
-
+from nflows.utils.torchutils import ensure_tensor
 from nflows.transforms.base import Transform
 
 
 class IdentityTransform(Transform):
     """Transform that leaves input unchanged."""
 
-    def forward(self, inputs, context=None):
-        batch_size = inputs.shape[0]
-        logabsdet = torch.zeros(batch_size)
+    def forward(self, inputs: torch.Tensor, context=Optional[torch.Tensor]):
+        num_batches = inputs.shape[0]
```

Comment:
We should probably stick to `batch_size` for consistency. Also,  `num_batches` doesn't make sense to me: we assume `inputs` will be a single batch?

Same in `PointwiseAffineTransform`.